### PR TITLE
DT-1158: ✅ Move user role test to cypress

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: integration-tests/integration/login/authuser.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    userPage.addRole().click()
+    cy.wait('@getUser')
+    const addRole = AuthUserAddRolePage.verifyOnPage('Auth Adm')
+
+    cy.task('stubAuthAddRole')
+    addRole.choose('Licence Vary')
+    cy.route({ method: 'GET', url: '/api/auth-user-roles-add?username=AUTH_ADM&role=LICENCE_VARY' }).as('addRole')
+    addRole.addRoleButton().click()
+
+    cy.wait('@addRole').should('have.property', 'status', 200)
```
Comment (petergphillips) [inline integration-tests/integration/login/authuser.spec.js:76]: this is quite cool, verifying response from xhr add role response is successful

File: manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/pages/AuthUserAddRolePage.groovy [hunk 2]
```diff
@@ -17,7 +17,6 @@ public class AuthUserAddRolePage extends Page {
     }
 
     void choose(String role) {
-        def option = selectOption { $("[data-qa='${role}_option']") }
```
Comment (petergphillips) [inline manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/pages/AuthUserAddRolePage.groovy:20]: not used

File: manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/specs/MaintainAuthUsersSpecification.groovy [hunk 3]
```diff
@@ -26,101 +26,6 @@ class MaintainAuthUsersSpecification extends BrowserReportingSpec {
 
     TestFixture fixture = new TestFixture(browser, prisonApi, oauthApi, tokenVerificationApi)
 
-    def "should allow a user search and display results"() {
```
Comment (petergphillips) [inline manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/specs/MaintainAuthUsersSpecification.groovy:29]: moved to cypress

**Review by JamieDraperUK — APPROVED** [review]
LGTM!

**Pull request merged**